pkg/cluster/cluster.go

```diff
@@ -243,6 +259,10 @@ type ClusterSpec struct {
 	ExistingConfig *ExistingConfig `json:"existingConfig,omitempty"`
 	// Standby setting when role is standby
 	StandbySettings *StandbySettings `json:"standbySettings,omitempty"`
+	// Set default hba auth for replication user and superuser.
+	// Values can be "all" or "standby", "all" allow access to all ips, "standby" restrict master access to standby servers ips.
+	// default is "all"
+	PgHBAAccess *PgHBAAccessMode `json:"pgHBAAccess,omitempty"`
```

Comment:
I'll call this `DefaultSUReplAccessMode`, longer but a bit clearer. Other Suggestions?

Reply:
We could use:
* DefaultSUReplAccessMode as option
* SUReplAccessMode as type
* DefaultSUReplAccess as default value
* SUReplAccessAll and SUReplAccessStrict as option names 
Is it ok for you ?

Reply:
Seems Good.